loky/backend/resource_tracker.py

```diff
@@ -13,15 +13,32 @@
 #
 # On Unix we run a server process which keeps track of unlinked
 # resources. The server ignores SIGINT and SIGTERM and reads from a
-# pipe.  Every other process of the program has a copy of the writable
-# end of the pipe, so we get EOF when all other processes have exited.
-# Then the server process unlinks any remaining resources.
-#
+# pipe. The resource_tracker implements a reference counting scheme: each time
+# a Python process anticipates the shared usage of a Python object by another
+# process, it signals the resource_tracker of this shared usage, and in return,
+# the resource_tracker increment the object's reference count by 1. Similarly,
```

Comment:
the resource's reference count

Resources are not necessarily Python objects: they are typically Operating System resources shared by several Python interpreters running in different Python processes.